riptide-spring-boot-autoconfigure/src/main/java/org/zalando/riptide/autoconfigure/ValueConstants.java

```diff
@@ -14,4 +14,6 @@ private ValueConstants() {}
     static final String METER_REGISTRY_REF = "\n\t\t\n\t\t\n\uE000\uE001\uE001\n\t\t\t\t\n";
     static final String TRACER_REF = "\n\t\t\n\t\t\n\uE000\uE001\uE002\n\t\t\t\t\n";
     static final String LOGBOOK_REF = "\n\t\t\n\t\t\n\uE000\uE001\uE003\n\t\t\t\t\n";
+    static final String TELEMETRY_TRACER_REF = "\n\t\t\n\t\t\n\uE000\uE001\uE004\n\t\t\t\t\n";
```

Comment:
We don't need this anymore, now we're getting the tracer from the `OpenTelemetry` instance.

Reply:
missed during clean up step